sdk/core/Azure.Core.TestFramework/src/RecordedTestBaseOfTEnvironment.cs

```diff
@@ -23,5 +27,11 @@ public override void StartTestRecording()
         }
 
         public TEnvironment TestEnvironment { get; }
+
+        [OneTimeSetUp]
+        public async ValueTask WaitForEnvironment()
+        {
+            await TestEnvironment.WaitForEnvironmentAsync();
```

Comment:
Can we add the same thing to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/src/LiveTestBase.cs#L11

Reply:
and https://github.com/kasobol-msft/azure-sdk-for-net/blob/dd244f41ee9c999dc09cb42cbd72930f1320770d/sdk/core/Azure.Core.TestFramework/src/SamplesBase.cs

Reply:
added in LiveTestBase, SamplesBase inherit from LiveTestBase.